katpoint/refraction.py

```diff
@@ -62,10 +62,22 @@ def refraction_offset_vlbi(el, temperature_C, pressure_hPa, humidity_percent):
     The code is based on poclb/refrwn.c in Field System version 9.9.2, which
     was added on 2006-11-15. This is a C version (with typos fixed) of the
     Fortran version in polb/refr.f. As noted in the Field System
-    documentation [Him1993]_, the refraction model originated with the Haystack
-    pointing system, but the documentation for this algorithm seems to have
-    been lost. It agrees well with the DSN refraction model, though.
+    documentation [Him1993b]_, the refraction model originated with the Haystack
+    pointing system. A description of the model can be found in [Clark1966]_,
+    which in turn references [IH1963]_ as the ultimate source.
 
+    References
+    ----------
+    .. [Him1993b] E. Himwich, "Station Programs," Mark IV Field System Reference
+       Manual, Version 8.2, 1 September 1993, available at
+       `<ftp://gemini.gsfc.nasa.gov/pub/fsdocs/stprog.pdf>`_
```

Comment:
That URL is giving me an unable to connect error.

Reply:
Yes, I realise that... This used to be the home of fsdocs but it's gone. I left it in as a hint to find the docs but the only recourse at the moment is to email the author.

I have copies of stprog.pdf and model.pdf. It might be sensible to include them in the katpoint documentation, given their relevance to our code.